specifyweb/frontend/js_src/lib/components/DataModel/resourceApi.ts

```diff
@@ -901,6 +901,8 @@ export const ResourceBase = Backbone.Model.extend({
       }
     });
 
+    if (self.specifyTable.name === 'CollectionRelationship') return json
```

Comment:
The solution seems very specific. It might be good to add a comment above the line to mention why this was needed